### ISSUE TITLE
Cannot use Shared Albums as Screensaver 

### ISSUE THREAD

**Issue by Robbbb** [chtaylo2/Roku-GooglePhotos#404]
I cannot use a shared album as the source for a screensaver. 
It would be great to be able to, so that the screensaver could automatically include new photos as they are contributed.

As it stands, I have to copy all the photos from a shared album into a personal one, but this is not a solution since I have to do it again each time new photos are added by a friend.

**Comment by chtaylo2** [comment]
Yes, you can. The channel supports specific personal albums and/or shared.
See here: https://www.photoviewapp.com/faq/#screensaver-album

**Comment by Robbbb** [comment]
Hey Chris!
Thanks for your prompt reply.
You are of course correct, thanks for the help, and for the amazing app. 

If it is helpful, this is how I came under the impression what I wanted was impossible:
My google account in question only has a album share with it, and has no albums of its own. When I initially booted up the screensaver, it said no photos found.
 I did not know that there were specific screensaver settings available in the complex Roku Settings > Theme > Screensaver > PhotoView etc interface.
 
From a UI perspective, I wonder if it would be possible to include a screensaver settings item inside of PhotoView, or perhaps a note explaining how to get to the screensaver settings menu, seeing as to me the screensaver settings lacked obvious discoverability.
